meson_options.txt

```diff
@@ -3,3 +3,6 @@ option('use_system_tzdb', type: 'boolean', value: false,
 
 option('use_tzdb_in_dot', type: 'boolean', value: false,
        description: 'Save the timezone database init the current folder')
+
+option('disable_string_view', type: 'boolean', value: false,
```

Comment:
Options should be named as "feature", not enable/disable.
See https://mesonbuild.com/Style-guide.html#naming-options

Could you please rename to `string_view` or something conforming to the style guide?